app/pages/collections/favorites-list.cjsx

```diff
@@ -8,7 +8,14 @@ FavoritesList = React.createClass
   title: 'Favorites'
 
   render: ->
-    props = Object.assign({}, @props, {favorite: true, translationObjectName:"favoritesPage"})
+    translationObjectName = "favoritesPage"
```

Comment:
app/pages/collections/collections-list.cjsx and app/pages/collections/favorites-list.cjsx  should be refactored to cut down on duplicated code.  A small module to return `{favorite, translationObjectName}` would probably do it.